### ISSUE TITLE
Non-compatible with Token Action HUD Core 2.0.4

### ISSUE THREAD

**Issue by BytomMan** [pwatson100/token-action-hud-symbaroum#5]
Upon loading Symbaroum tells me that the Token Action HUD Symbaroum RPG will only work with Token Action HUD Core 1.5. Core 2.0.4 is the latest version.

**Comment by pwatson100** [comment]
That is correct.  The latest version of TAH Core has a number of breaking changes and is not compatable with older versions of our modules.  You will need to downgrade to the last version of TAH Core untill I have a chance to asses and make the required changes.

**Comment by pwatson100** [comment]
New version updated to work with TAH V2
